src/lib/components/Snackbar.react.js

```diff
@@ -41,6 +42,8 @@ type Props = {
   },
   /** CSS class name of the root element */
   className?: string,
+  /** Elements to render inside the snackbar. Note that this does not work along message and actions */
+  children?: Node,
```

Comment:
Phrasing here is a bit odd.

Reply:
Not sure what I meant here, clearly not correct. I clarified now